integration-tests/pom.xml

```diff
@@ -51,7 +57,22 @@
         <dependency>
             <groupId>org.apache.cxf</groupId>
             <artifactId>cxf-rt-transports-http-netty-server</artifactId>
+            <version>${cxf.version}</version>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>jakarta.activation</groupId>
+                    <artifactId>jakarta.activation-api</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>jakarta.xml.bind</groupId>
+                    <artifactId>jakarta.xml.bind-api</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.geronimo.specs</groupId>
+                    <artifactId>geronimo-jta_1.1_spec</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
These exclusions are already in the BOM, no?

Reply:
No, cxf-rt-transports-http-netty-server is not in the BOM. It is a test dependency and it does not belong there, IMO. I'd like to remove it altogether in near future and replace it with a testcontainer.